client/src/components/App.jsx

```diff
@@ -1,21 +1,76 @@
 import React, { Component } from 'react';
 import axios from 'axios';
+import styled from 'styled-components';
+import {ReserveButton, Box, Line, GeneralText, DatesBox} from '../styling/reactStyles.js';
+import Price from './Price.jsx';
+import Reviews from './Reviews.jsx';
+
+
 
 
 class App extends Component {
   constructor(props) {
     super(props);
     this.state = {
-      listingInfo: []
+      listingInfo: [],
+      id: '',
+      reservedDates: [],
+      month: ''
     };
-    this.initialize = this.initialize.bind(this)
+    this.initialize = this.initialize.bind(this);
+    this.getCurrentCalendar = this.getCurrentCalendar.bind(this);
+    this.getNextCalendar = this.getNextCalendar.bind(this);
+    this.getPreviousCalendar = this.getPreviousCalendar.bind(this);
   }
 
+  
+
   initialize() {
```

Comment:
Is this intended to be used like componentDidMount? Or where/when would it be called?

Reply:
When components mount, initialize is intended to pull information about one random listing and change App's state with that information. When we merge our components, it will need to be modified to pull information about a specific listing.

Reply:
Ahh, sorry didn't see the componentDidMount below (github hid the unchanged lines), which calls this method. Makes more sense now, thanks!